README.md

```diff
@@ -287,53 +287,161 @@ Although this example is pretty simple and not even includes Lens composition, i
 
 Just as with React, but assuming Angular instead :)
 
+The code for below example - can be found here <https://github.com/georgebatalinski/telescope_todo>
+
 As a bootstrap for the application we need something like:
 
+```typescript 
+@NgModule({
+  declarations: [
+    AppComponent,
+    TodoListComponent
+  ],
+  imports: [
+    ...
+  ],
+  providers: [
+    { provide: 'StreamContract', useFactory: ()=> Telescope.of( AppState.empty() ), deps: [] }
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+We define our global STATE (however, the more ultimate approach would be to have local state)
+local state = would mean each Service, would have a Telescope instance of their own, resulting in us not 
+having to setup and/or maintain <AppState> 
+Note: If choose to maintain <AppState> as in this example, we would be adding more members to the 
```

Comment:
Consider writing an alternative solution after the example. Adding notes before the sample code can distract the reader.

Reply:
Maybe we should do a seperate post for this to add to documentation? But keep the initial example - since it was much better for Getting Started